boringtun/src/device/integration_tests/mod.rs

```diff
@@ -482,7 +482,7 @@ mod tests {
     fn test_wireguard_set() {
         let port = next_port();
         let private_key = StaticSecret::random_from_rng(OsRng);
-        let own_public_key = PublicKey::from(&private_key);
+        let _own_public_key = PublicKey::from(&private_key);
```

Comment:
:thinking:  maybe it is worth removing it this, if `own_public_key` is not used?

Reply:
Done